src/Uno.Extensions.Toolkit.UI/common.props

```diff
@@ -0,0 +1,28 @@
+<Project ToolsVersion="15.0">
+	<PropertyGroup>
+		<AssemblyName>Uno.Extensions.Toolkit.UI</AssemblyName>
+		<GenerateLibraryLayout>true</GenerateLibraryLayout>
+		<RootNamespace>Uno.Extensions.Toolkit</RootNamespace>
+		<!--Temporary disable missing XML doc until fixed in the whole package-->
+		<WarningsNotAsErrors>$(WarningsNotAsErrors);CS1591</WarningsNotAsErrors>
+		<!-- As we are InternalsVisibleTo, we disable some compatibility types that are not used by package itself. -->
+		<UnoExtensionsGeneration_DisableModuleInitializerAttribute>True</UnoExtensionsGeneration_DisableModuleInitializerAttribute>
+	</PropertyGroup>
+	<ItemGroup>
+		<ProjectReference Include="..\Uno.Extensions.Toolkit\Uno.Extensions.Toolkit.csproj" />
+	</ItemGroup>
+
+
+	<ItemGroup>
+		<Page Include="**\*.xaml" Exclude="bin\**\*.xaml;obj\**\*.xaml" />
+		<Compile Update="**\*.xaml.cs">
+			<DependentUpon>%(Filename)</DependentUpon>
+		</Compile>
+		<UpToDateCheckInput Include="**\*.xaml" Exclude="bin\**\*.xaml;obj\**\*.xaml" />
+	</ItemGroup>
```

Comment:
let's exclude this for now as we don't have any xaml we currently need to deal with.